lib/jekyll-crds/filters/media_url_filter.rb

```diff
@@ -0,0 +1,21 @@
+module Jekyll
+  module MediaUrlFilter
+
+    def media_url(item, arg = nil)
+      # binding.pry
+      if item.nil? 
+        return
+      end
+
+      item["bitmovin_url"].nil? ? item.url : append_video_params(item, arg)
```

Comment:
1). Remove the `pry` comment. 

2). Maybe wrap the whole method in the condition and loose the return altogether?

```ruby
def media_url(item, arg = nil)
  unless item.nil?
    ...
  end
end
```

Reply:
i recall you not liking the `unless` but perhaps your icy heart has found some warmth in this weary world.

will change both things

Reply:
Done

Reply:
Ok, that's fair criticism. 

I have argued in the past that `unless` is less readable than `if` since it 
negates the condition. I can usually tolerate `unless` as long as the context is clear, as it is in this case. 

Mostly, I prefer the inline syntax, as such... 

```ruby
do_something unless condition?
```